test/google/gax/operation_test.rb

```diff
@@ -28,12 +28,9 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 require 'test_helper'
-require 'time'
-
+# require 'time'
```

Comment:
Is this intentional or meant to be removed?

Reply:
I was checking if it was needed. The only methods being used is `Time.now`, which does not need the require. The require adds `Time.parse`, which is not being used.

Removed.